generators/resource/templates/actions/resource-use_model_test.go.tmpl

```diff
@@ -5,8 +5,138 @@ import (
 
 	"github.com/stretchr/testify/require"
 )
-{{ range $a := .actions }}
-func (as *ActionSuite) Test_{{$.opts.Name.Resource}}Resource_{{ camelize $a }}() {
-	as.Fail("Not Implemented!")
+
+func (as *ActionSuite) TableChange(table string, count int, fn func()) {
```

Comment:
Should we check if this function already exists somewhere ? otherwise we would have this function repeated for every resource we generate causing that the code doesn't compile.